packages/sit-onyx/src/composables/useCustomValidity.ts

```diff
@@ -207,6 +210,22 @@ export const useCustomValidity = (options: UseCustomValidityOptions) => {
     };
   });
 
+  const successMessages = computed<FormMessages | undefined>(() => {
+    if (validityState.value === undefined || !validityState.value.valid) return;
+
+    const errorType = getFirstInvalidType(validityState.value);
+    const customErrors = getCustomMessages(options.props.customError);
+
+    // a custom error message always is considered first
+    if (customErrors || errorType === "customError") {
+      if (customErrors) return;
+    }
```

Comment:
```suggestion
```

Why is this needed? If custom errors would exist `validatyState.value-valid` would be false which is already covered in the if above :)

Reply:
yes, good point here :)